aws_lambda_powertools/utilities/batch/base.py

```diff
@@ -94,7 +98,7 @@ def __enter__(self):
     def __exit__(self, exception_type, exception_value, traceback):
         self._clean()
 
-    def __call__(self, records: List[dict], handler: Callable):
+    def __call__(self, records: List[dict], handler: Callable, lambda_context: Optional[LambdaContext] = None):
```

Comment:
note to self: take the opportunity to make `records` typing more specific. Need to check with Mypy accepts it due to liskov substitution principle.

Reply:
Since I can access record properties in my Lambda code, I'm not sure this is a real concern for now. Maybe in the future it's worth spending some time on it?
But yes, I love static and strong typing :heart: